app/src/main/AndroidManifest.xml

```diff
@@ -200,7 +203,8 @@
 
```

Comment:
`.asynchronous.services.ftp.FtpService` would also benefit from having the permission